src/websocket/app.ts

```diff
@@ -49,14 +49,14 @@ export class AppWebsocket implements AppApi {
     = this._requester('zome_call_invocation', callZomeTransform)
 }
 
-const callZomeTransform: Transformer<CallZomeRequestGeneric<any>, CallZomeRequestGeneric<Buffer>, CallZomeResponseGeneric<Buffer>, CallZomeResponseGeneric<any>> = {
-  input: (req:  CallZomeRequestGeneric<any>): CallZomeRequestGeneric<Buffer> => {
+const callZomeTransform: Transformer<CallZomeRequestGeneric<any>, CallZomeRequestGeneric<Uint8Array>, CallZomeResponseGeneric<Buffer>, CallZomeResponseGeneric<any>> = {
```

Comment:
The `CallZomeResponseGeneric` still has a `Buffer` in it.  Not sure that's right.

Reply:
Yeap thanks, fixed it. Actually I think the whole library would benefit from removing Buffer as one of its core main types, as this is mainly going to be used in UIs.